src/features/CareViewPatients/components/CareViewPatients.jsx

```diff
@@ -53,13 +89,54 @@ export const CareViewPatients = () => {
     setLimit(e.pageSize);
   };
 
+  const handleSearchOnChange = (e) => {
+    updateSearchValue(e.target.value);
+  };
+
+  const handleKeyPress = (event) => {
+    if (event.key === "Enter") {
+      if (searchValue.length >= 3) {
+        setCurrentPage(1);
+        setIsSearched(true);
+        getPatientsListBySearch();
+      } else {
+        setIsSearched(false);
+      }
+    }
+  };
+  const handleClear = () => {
+    updateSearchValue("");
+    setIsSearched(false);
+    if (selectedWard.value) {
+      getPatientsList();
+    }
+  };
+
+  const noSearchResultsForPatientsWardMessage = (
+    <FormattedMessage
+      id={"NO_SEARCH_RESULTS_PATIENTS_WARD_MESSAGE"}
+      defaultMessage={"No search result available"}
+    />
+  );
+
```

Comment:
Can we create a careviewPatientsHeader component and move search related thing to the that component?

Reply:
Sure Ayush.